server/templates/email/media-request/html.pug

```diff
@@ -25,93 +25,60 @@ head
-          tr
-            td(align='center' style='\
-            padding-top: 25px;\
-            padding-bottom: 25px;\
-            text-align: center;\
-            ')
-              a(href=applicationUrl style='\
-              text-shadow: 0 1px 0 #ffffff;\
-              font-weight: 700;\
-              font-size: 24px;\
-              color: #a8aaaf;\
-              text-decoration: none;\
-              ')
-                | #{applicationTitle}
-          tr
-            td(style='width: 100%' width='100%')
-              table.sm-w-full(align='center' style='\
-              background-color: #ffffff;\
-              margin-left: auto;\
-              margin-right: auto;\
-              width: 570px;\
-              ' width='570' bgcolor='#ffffff' cellpadding='0' cellspacing='0' role='presentation')
-                tr
-                  td(style='padding: 45px')
-                    div(style='font-size: 16px')
-                      | #{body}
-                      br
-                      br
-                      p(style='margin-top: 4px; text-align: center')
-                        b
-                          | #{mediaName}
-                        each extra in mediaExtra
-                          br
-                          | #{extra.name}:&nbsp;
-                          | #{extra.value}
-                      table(align='center' cellpadding='0' cellspacing='0' role='presentation')
-                        tr
-                          td
-                            a(href=actionUrl style='color: #3869d4')
-                              img(src=imageUrl alt='')
-        p(style='\
-        font-size: 16px;\
-        line-height: 24px;\
-        margin-top: 6px;\
-        margin-bottom: 20px;\
-        color: #51545e;\
-        ')
-          | Requested by #{requestedBy} at #{timestamp}
-        p(style='\
-        font-size: 13px;\
-        line-height: 24px;\
-        margin-top: 6px;\
-        margin-bottom: 20px;\
-        color: #51545e;\
-        ')
-          a(href=actionUrl style='color: #3869d4') Open in #{applicationTitle}
-tr
-  td
-    table.sm-w-full(align='center' style='\
-    margin-left: auto;\
-    margin-right: auto;\
-    text-align: center;\
-    width: 570px;\
-    ' width='570' cellpadding='0' cellspacing='0' role='presentation')
+      td
+        div(style='box-sizing: border-box; margin: 0; width: 100%; color: #fff; border-radius: .75rem; padding: 1rem; border: 1px solid rgba(100, 100, 100, 1); background: linear-gradient(135deg, rgba(17, 24, 39, 0.47) 0%, rgb(17, 24, 39) 75%), url(' + imageUrl + ') center 25%/cover')
+          table(style='color: #fff; width: 100%;')
+            tr
+              td(style='vertical-align: top; font-size: 1.125rem; line-height: 1.75rem;')
+                a(href=actionUrl style='display: block; max-width: 16rem; overflow: hidden; color: #fff; text-overflow: ellipsis; white-space: nowrap;')
```

Comment:
It might make sense to wrap text instead of truncating it with ellipsis here, since otherwise any titles with more than 3-4 words would not be fully visible.  You could also make the card wider?  I feel like we should definitely show more / truncate less than in your PR description screenshot though.  The email doesn't have the same space constraints as the Overseerr app, so while it's inspired by the app, it doesn't need to match _exactly_. 😄

That said, bolding the media title and only underlining it on hover would more closely match the styling of the request cards in the Overseerr app.

I'm really just nitpicking now 😝  This really looks great!  Would be nice to restyle the other email templates as well, after the fixes in #1806 are in.

Reply:
I think these are great suggestions! I will see about getting these changes in and tested locally. I do want to style the other email notifications, but it will definitely be easier once those changes in #1806 go into place.

Reply:
![image](https://user-images.githubusercontent.com/25516090/124686294-b5fadd00-de87-11eb-9676-d01a021b6aca.png)